.changeset/early-knives-perform.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-msgraph': patch
```

Comment:
as this is changing the name of the generated entities, not sure if this is a just a patch change

Reply:
Good question 😄 . I can make it a minor one.